cel/io.go

```diff
@@ -126,6 +128,55 @@ func ValueAsAlphaProto(res ref.Val) (*exprpb.Value, error) {
 	return alpha, err
 }
 
+// RPCStatusToEvalErrorStatus converts an RPC status to a celpb status. Status value is intended to
+// be wire and field compatible with `google.rpc.Status`.
+func RPCStatusToEvalErrorStatus(s *rpcpb.Status) (*celpb.Status, error) {
+	var celpbStatus celpb.Status
+	statusVal, err := proto.Marshal(s)
+	if err != nil {
+		return nil, err
+	}
+	err = proto.Unmarshal(statusVal, &celpbStatus)
+	if err != nil {
+		return nil, err
+	}
+	return &celpbStatus, nil
+}
+
+// RefValToExprValue converts between ref.Val and cel.expr.ExprValue.
+// The result ExprValue is the serialized proto form.
+func RefValToExprValue(res ref.Val) (*celpb.ExprValue, error) {
+	switch res := res.(type) {
+	case *types.Unknown:
+		return &celpb.ExprValue{
+			Kind: &celpb.ExprValue_Unknown{
+				Unknown: &celpb.UnknownSet{
+					Exprs: res.IDs(),
+				},
+			}}, nil
+	case *types.Err:
+		s := status.Convert(res.Unwrap()).Proto()
```

Comment:
Please don't take a direct dependency on grpc status here, as this is intended for gRPC APIs to propagate status up to proto APIs. Instead, just convert the error to a CEL status error proto.